### DISCUSSION TITLE
PyHanko CLI addsign pemder gives me an error

### DISCUSSION THREAD

**Discussion by Ziad-Assem** [MatthiasValvekens/pyHanko#274] [Q&A]
When I run the following cli code:
`pyhanko sign addsig --no-strict-syntax --field Sig1 pemder --key private_key.pem --cert certificate.der gg.pdf output.pdf --no-pass`

i get the following error:  **pyhanko.cli - ERROR - Generic processing error. Error: Generic processing error.**

PDF is Spec. Ver 1.7

-I'm trying to digitally sign a PDF using PKCS#7. I used the following bat code to generate the certificate and private key:

private key:
`openssl genpkey -algorithm rsa -out private_key.pem -pkeyopt rsa_keygen_bits:2048`

certificate:
`openssl req -new -x509 -sha256 -key private_key.pem -outform der -out certificate.der -days 365`

**Comment by MatthiasValvekens** [comment]
Hi @Ziad-Assem 


Run with `pyhanko --verbose` and post the stack trace, and preferably also the input file (if you can), since this could be due to pretty much anything :)
